waspc/packages/deploy/src/providers/fly/setup/setup.ts

```diff
@@ -0,0 +1,79 @@
+import { $, cd } from 'zx';
+import crypto from 'crypto';
+import {
+	clientTomlExistsInProject,
+	copyLocalClientTomlToProject,
+	copyLocalServerTomlToProject,
+	deleteLocalToml,
+	getTomlFilePaths,
+	replaceLineInLocalToml,
+	serverTomlExistsInProject,
+} from '../helpers/tomlFileHelpers.js';
+import { createDeploymentInfo, DeploymentInfo } from '../DeploymentInfo.js';
+import { GlobalOptions } from '../GlobalOptions.js';
+import { cdToClientBuildDir, cdToServerBuildDir, makeIdempotent, getCommandHelp, waspSays } from '../helpers/helpers.js';
+import { createFlyDbCommand } from '../index.js';
+
+export async function setup(baseName: string, region: string, options: GlobalOptions): Promise<void> {
+	waspSays('Setting up your Wasp app with Fly.io!');
+
+	const buildWasp = makeIdempotent(async () => {
+		waspSays('Building your Wasp app...');
+		cd(options.waspProjectDir);
+		await $`${options.waspExe} build`;
+	});
+
+	const tomlFilePaths = getTomlFilePaths(options);
+	const deploymentInfo = createDeploymentInfo(baseName, region, options, tomlFilePaths);
+
+	if (serverTomlExistsInProject(tomlFilePaths)) {
+		waspSays(`${tomlFilePaths.serverTomlPath} exists. Skipping server setup.`);
+	} else {
+		await buildWasp();
+		await setUpServer(deploymentInfo);
+	}
+
+	if (clientTomlExistsInProject(tomlFilePaths)) {
+		waspSays(`${tomlFilePaths.clientTomlPath} exists. Skipping client setup.`);
+	} else {
+		await buildWasp();
+		await setupClient(deploymentInfo);
+	}
+
+	waspSays(`Don't forget to create your database by running "${getCommandHelp(createFlyDbCommand)}".`);
+}
+
+async function setUpServer(deploymentInfo: DeploymentInfo) {
```

Comment:
Nit: `setUp` vs `setup` in `setupClient`

Reply:
I will change `setUpServer` back to `setupServer` for consistency with all the other `setup` ones. Sorry about that @sodic all lowercase just feels a bit better to me 😅

Reply:
Nobody respects that you are the only native speaker here @shayneczyzewski :D.

Reply:
Missed this comment.

I mean ok, sure, if you guys want to be **WRONG** then yeah, fine, cool, great. 

<details><summary>...</summary> 

I am joking of course, but "set up" truly [is correct in this case](https://grammarist.com/spelling/set-up-vs-setup/#:~:text=Set%20Up%20%E2%80%93%20What's%20the%20Difference,set%20up%20the%20computer%E2%80%9D) :upside_down_face:.
</details>